libs/boringssl/Makefile

```diff
@@ -28,7 +28,7 @@ define Package/boringssl
 	CATEGORY:=Libraries
 	TITLE:=An implementation of the TLS protocol
 	URL:=https://boringssl.googlesource.com/boringssl/
-	DEPENDS:=+libstdcpp $(GO_ARCH_DEPENDS) @!(mips||mips64)
```

Comment:
Well, `$(GO_ARCH_DEPENDS)` doesn't include any runtime dependencies actually, but just limit the package to build for architectures supported by golang only.